lib/runner.ex

```diff
@@ -0,0 +1,39 @@
+defmodule Runner do
+  alias Console
+  alias TTT
+  alias Messages
+
+  def main(_args) do
+    start_game()
+  end
+
+  def start_game() do
+    TTT.start_new_game(Console.get_user_input(Messages.get_message(:start_game)))
+    |> begin_game?()
+
+    get_markers()
+  end
+
+  def begin_game?("Y"), do: Console.display_message(Messages.get_message(:welcome))
+  def begin_game?("N"), do: Console.display_message(Messages.get_message(:goodbye))
+
+  def begin_game?(_) do
+    Console.display_message(Messages.get_message(:choose_again))
```

Comment:
Here I think you can do more piping. I might be off here since it has been a while since I've done this:
``` 
          :choose_again
          |> Messages.get_message
          |> Console.display_message
```

Reply:
Cool, makes sense! I've got it working :)